sanic/config.py

```diff
@@ -1,10 +1,75 @@
+from sanic.defaultFilter import DefaultFilter
 import os
-
 import types
 
 SANIC_PREFIX = 'SANIC_'
 
 
+DEFAULT_LOG_CONF = {
```

Comment:
Not really. But it should not be named with prefix `DEFAULT_` as it is a config, not just default.

`LOGGING_CONF` would be fine but it is already in `config` module so `LOGGING` makes more sense.